recipes/spades/meta.yaml

```diff
@@ -1,18 +1,19 @@
+{% set version = "3.11.0" %}
+
 package:
   name: spades
-  version: '3.10.1'
+  version: {{ version }}
 
 source:
-  fn: SPAdes-3.10.1-Linux.tar.gz                                              # [linux]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Linux.tar.gz  # [linux]
-  md5: fb58d48a0545211aab91f43b7d17ad1b # [linux]
-  fn: SPAdes-3.10.1-Darwin.tar.gz                                             # [osx]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Darwin.tar.gz # [osx]
-  md5: f23bbad7921cff2c283e6589ccee83a2 # [osx]
+  fn: SPAdes-{{ version }}-Linux.tar.gz                                              # [linux]
+  url: http://spades.bioinf.spbau.ru/release{{ version }}/SPAdes-{{ version }}-Linux.tar.gz  # [linux]
+  sha256: 5d362ea40bfc0fedfbdf7d17504f38f63e7e7e23b29fef89f89273313115db5f # [linux]
+  fn: SPAdes-{{ version }}-Darwin.tar.gz                                             # [osx]
+  url: http://spades.bioinf.spbau.ru/release{{ version }}/SPAdes-{{ version }}-Darwin.tar.gz # [osx]
+  sha256: 9e6accf2258c379436e92f37f920e4dc8fa4a2da9a9813215709a9723bf15d87 # [osx]
```

Comment:
Downloading binaries for spades has been working fine.  But in general, I remember being told that there's a preference for building things from source?  ( What are the exact reasons for preferring builds from source?  @bgruening @johanneskoester Maybe clarify this in https://bioconda.github.io/guidelines.html ? )